python/pyspark/sql/tests/test_functions.py

```diff
@@ -1072,6 +1072,44 @@ def test_binary_math_function(self):
         for a, e in zip(df.first(), expected):
             self.assertAlmostEqual(a, e, 5)
 
+    def test_map_functions(self):
+        from pyspark.sql import functions as F
```

Comment:
Let's move this import to the top, and add a comment like:

```python
def test_case(self):
    # SPARK-12345: a short description of the test
    ...
```

See also https://spark.apache.org/contributing.html